### ISSUE TITLE
module 'tensorflow.math' has no attribute 'is_nan'

### ISSUE THREAD

**Issue by fancyfeifei** [bubbliiiing/mask-rcnn-keras#19]
十分感谢您的付出！
我是按照说明使用tensorflow1.13.1，但是开始训练时报错了AttributeError: module 'tensorflow.math' has no attribute 'is_nan'

**Comment by bubbliiiing** [comment]
嗯？一般来讲不会有这个问题，你是不是选错环境了
